packages/core/src/common/keybinding.ts

```diff
@@ -53,6 +60,88 @@ export namespace KeybindingContexts {
     };
 }
 
+export class DefaultKeybindingIndex {
+    protected readonly defaultBindings: { [index: string]: Keybinding[] } = {};
+    protected readonly defaultCommands: { [commandId: string]: Keybinding[] } = {};
+
+
+    constructor() {
```

Comment:
I do not see a reason for the empty `ctor` function. So we could get rid of the one in the sub-class too.